js/articles.js

```diff
@@ -0,0 +1,71 @@
+fetch('jsons/articles.json')
+  .then(response => {
+    return response.json()
+  })
+  .then(data => {
+    // Work with JSON data here
+    articlesCounter = data["articles"].length;
+    let a = Math.floor(Math.random() * articlesCounter);
```

Comment:
first of all you don't change these variables - make them constants
secondly, rename like firstArticle, secondArticle and so on
or create an array like randomArticles and then go throw it